src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework19 extends BaseTest {
+    //
+    @Test
+    public void deletePlaylist()throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
got it thank you